python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -339,7 +339,7 @@ def processResults(self, coaddExposure, dataRef):
             )
             try:
                 psfResults = self.measurePsf.run(coaddExposure, coaddSources, expId=expId)
-            except RuntimeError as e:
+            except Exception as e:
```

Comment:
I agree it's ugly to catch *all* exceptions like this, but I think that's what we actually want here. We don't usually re-measure the coadd PSF, so this is an additional step to try to improve accuracy. But, if it fails for any reason there's still a perfectly good PSF model to fall back on, and it seems excessive to make the entire task fail in that case.

Reply:
Okay.  You should probably add a comment here explaining this, in case some enterprising future version of ourselves has this same discussion and concludes differently.